CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
```

Comment:
Sweet! Nice idea to keep track of changes 👏

Reply:
> @hramezani do you think we should bump the major version?

Sure! 

Before release, do we want to drop support for Python < 3.6?